.github/workflows/pytest.yaml

```diff
@@ -0,0 +1,36 @@
+name: python-tests
+
+on:
+  pull_request:
+    types:
```

Comment:
Do those all need to be listed ? I thought if you leave the list empty, then it applies to _all_ PRs. If we have a reason to enumerate them that's fine, but otherwise it just looks like a bit of clutter.